projects/frontend/data-pipelines/gui/projects/ui/src/app/app.component.spec.ts

```diff
@@ -19,25 +19,55 @@ import {
     UrlOpenerService
 } from '@versatiledatakit/shared';
 
-import { AppConfig } from './app-config.model';
-
 import { AppConfigService } from './app-config.service';
 
 import { AppComponent } from './app.component';
 
 describe('AppComponent', () => {
     let routerServiceStub: jasmine.SpyObj<RouterService>;
-    let oAuthServiceStub: jasmine.SpyObj<OAuthService>;
     let navigationServiceStub: jasmine.SpyObj<NavigationService>;
     let viewContainerRefStub: jasmine.SpyObj<ViewContainerRef>;
     let dynamicComponentsServiceStub: jasmine.SpyObj<DynamicComponentsService>;
     let confirmationServiceStub: jasmine.SpyObj<ConfirmationService>;
     let urlOpenerServiceStub: jasmine.SpyObj<UrlOpenerService>;
     let appConfigServiceStub: jasmine.SpyObj<AppConfigService>;
+    const configureTestingModule = (providersAdditional: any[]) => {
```

Comment:
You could use here for type `Provider[]`

Reply:
Most of the linters are configured to WARN when any type is used.

Reply:
Yup, I'm familiar with a Provider (I've browsed the data model before to understand it better).
However, the field providers that we do assign to, is of type any[], that's the contract.